docs/en/data_types/datetime64.md

```diff
@@ -0,0 +1,60 @@
+# DateTime64 {#data_type-datetime64}
+
+Stores a moment in time, like DateTime, but with 2 differences:
```

Comment:
We'd better describe it from scratch, without referencing DateTime

Reply:
Done